client/my-sites/importer/uploading-pane.jsx

```diff
@@ -16,8 +17,9 @@ import Button from 'components/forms/form-button';
 import DropZone from 'components/drop-zone';
 import ProgressBar from 'components/progress-bar';
 import Gridicon from 'components/gridicon';
+import { connectDispatcher } from './dispatcher-converter';
 
-export default React.createClass( {
+export const UploadingPane = React.createClass( {
```

Comment:
It should be fine to leave as is if you'd like a shorter local name. We'd just need to remember in tests to `import { UploadingPane } from 'my-sites/importer/uploading-pane'`.